docs/initrd.md

```diff
@@ -0,0 +1,10 @@
+To build an image with a mkosi-built initrd:
```

Comment:
Let's add a title here: `# Building a custom initrd and using it in a mkosi image`